packages/react-components/react-tabster/package.json

```diff
@@ -35,7 +35,7 @@
     "@fluentui/react-shared-contexts": "^9.0.0",
     "@fluentui/react-theme": "^9.0.0",
     "@fluentui/react-utilities": "^9.0.2",
-    "keyborg": "^1.1.0",
+    "keyborg": "^1.1.1",
```

Comment:
why not bump to latest `^1.2.1` ?

Reply:
yup, done in https://github.com/microsoft/fluentui/pull/24376/commits/a9f763511c4f0fc61c21155d90ad2f4adc807d57